packages/formation/js/additional-info.js

```diff
@@ -43,22 +49,28 @@ export default function createAdditionalInfoWidget() {
       const chevron = qS(el, 'i.fa-angle-down');
       const button = qS(el, 'button');
 
-      const additionalInfoClickEvent = document.createEvent('Event');
-      additionalInfoClickEvent.initEvent(
-        '@department-of-veterans-affairs/formation/additional-info/button-clicked',
-        true,
-        true,
-      );
-      additionalInfoClickEvent.detail = { titleText, dataset };
+      const additionalInfoExpandEvent = document.createEvent('Event');
```

Comment:
If I'm not mistaken, I think MDN prefers devs using `new Event()` over `createEvent` as some browsers may in the near-future deprecate those methods. https://developer.mozilla.org/en-US/docs/Web/API/Event/initEvent

Reply:
Unfortunately, `new Event` isn't supported by IE - https://developer.mozilla.org/en-US/docs/Web/API/Event/Event - so we're stuck with this method 😞

Reply:
Oh wow, such conflicting info from MDN then... 😂Thanks IE! ✊